app/src/main/java/com/kickstarter/ui/activities/DeepLinkActivity.java

```diff
@@ -0,0 +1,79 @@
+package com.kickstarter.ui.activities;
+
+import android.content.Intent;
+import android.net.Uri;
+import android.os.Bundle;
+import android.os.Parcelable;
+import android.support.annotation.Nullable;
+import android.text.TextUtils;
+
+import com.kickstarter.libs.BaseActivity;
+import com.kickstarter.libs.RefTag;
+import com.kickstarter.libs.qualifiers.RequiresActivityViewModel;
+import com.kickstarter.libs.utils.ApplicationUtils;
+import com.kickstarter.ui.IntentKey;
+import com.kickstarter.viewmodels.DeepLinkViewModel;
+
+import java.util.List;
+
+import static com.kickstarter.libs.rx.transformers.Transformers.observeForUI;
+
+@RequiresActivityViewModel(DeepLinkViewModel.ViewModel.class)
+public class DeepLinkActivity extends BaseActivity<DeepLinkViewModel.ViewModel> {
+  @Override
+  protected void onCreate(@Nullable final Bundle savedInstanceState) {
```

Comment:
we like to put our annotations right next to what it's annotating, e.g. `final @Nullable Bundle savedInstanceState`